src/Classes/MyRadio/CoreUtils.php

```diff
@@ -554,6 +554,34 @@ public static function getAllPermissions()
     }
 
     /**
+     * udiff function for permission value equality
+     * @param  array $perm1 permission value & description
+     * @param  array $perm2 permission value & description
+     * @return int          comparison result
+     */
+    private static function comparePermission($perm1, $perm2)
```

Comment:
Thinking about #319, should these potentially be moved out in that one? In which case if this is merged should that PR be updated to reflect it?